dynamic_rest/filters.py

```diff
@@ -130,16 +137,46 @@ def generate_query_key(self, serializer):
             if isinstance(s, serializers.ListSerializer):
                 s = s.child
             if not s:
-                raise ValidationError(
-                    "Invalid nested filter field: %s" % field_name
-                )
+                raise ValidationError("Invalid nested filter field: %s" % field_name)
 
         if self.operator:
             rewritten.append(self.operator)
 
         return ('__'.join(rewritten), field)
 
 
+def rewrite_filters(fs, serializer):
+    out = {}
+    for node in fs.values():
+        filter_key, field = node.generate_query_key(serializer)
+        if isinstance(field, (BooleanField, NullBooleanField)):
+            node.value = is_truthy(node.value)
+        out[filter_key] = node.value
+
+    return out
+
+
+def clause_to_q(clause, serializer):
+    key, value = clause
+    negate = False
+    q = {}
+    if key.startswith('-'):
+        negate = True
+        key = key[1:]
+    parts = key.split('.')
+    operator = 'eq'
+    if parts[-1] in DynamicFilterBackend.VALID_FILTER_OPERATORS:
+        operator = parts.pop()
+    if operator == 'eq':
+        operator = None
```

Comment:
why not just initialize this to `None`

Reply:
In case it was explicitly set to `eq` by the user, its still better to leave it off because some types don't work correctly with `eq` (like UUID)